pandas/tests/extension/test_numpy.py

```diff
@@ -330,17 +330,6 @@ def test_fillna_frame(self, data_missing):
         # Non-scalar "scalar" values.
         super().test_fillna_frame(data_missing)
 
-    def test_fillna_fill_other(self, data_missing):
```

Comment:
why's this removed?

Reply:
calling ._values here https://github.com/pandas-dev/pandas/pull/40758/files#diff-1a2e3df0db7dd8bddc2ec4bff9de8a7a55e328e6c32e2cecde761dc9549fcd46R3544 gives us an ndarray whereas ATM we get a PandasArray, so we no longer need to override the parent class's test